src/GeositeFramework/region.json

```diff
@@ -8,7 +8,8 @@
         "url": "http://www.azavea.com/"
     },
     "headerLinks": [
-        { "text": "Menu", "url":"#", "items": [
+       { "text": "Getting started", "launchpad": true, "launchpadId": 1, "url": "javascript:;" },
```

Comment:
`url` is a required parameter. I could make it not required, but I'm not totally sure that we want to do that.